src/main/webapp/app/overview/course-conversations/layout/conversation-selection-sidebar/conversation-sidebar-section/conversation-sidebar-section.component.html

```diff
@@ -15,7 +15,7 @@
         }
     </div>
     <div [(ngbCollapse)]="isCollapsed">
-        @if (visibleConversations.length > 0 || hiddenConversations.length > 0) {
+        @if (visibleConversations.length > 0 || mutedConversations.length > 0 || hiddenConversations.length > 0) {
```

Comment:
You can simple check for length
  @if (visibleConversations.length || mutedConversations.length || hiddenConversations.length)

Reply:
Good idea 🙂